public/js/admin.js

```diff
@@ -383,10 +437,15 @@ var updateVisibility = function(qid) {
         },
         success: function(data) {
             if (data == 'failure') {
-		displayQuestionTable();
+                displayQuestionTable();
```

Comment:
Indentation is off here.

Reply:
It was actually off before, and I fixed it that way. Let me know if I am wrong, because my editor is messed up when it comes to spacing XD